### ISSUE TITLE
Application reference and snippets not working in "ixpeobssim.readthedocs"

### ISSUE THREAD

**Issue by lucabaldini** [lucabaldini/ixpeobssim#620]
Looking at online documentation for ixpeobssim, the “application reference“ does not list the command help list, as it should.

(see https://ixpeobssim.readthedocs.io/en/latest/reference.html )

Similarly, snippets are not working in subsections “Model components“ and “Region of Interest“.

(see https://ixpeobssim.readthedocs.io/en/latest/source_models.html)

**Comment by lucabaldini** [comment]
Ok---this was more complicated than I anticipated, and it is due to the fact that the environment we have on RTD is not the same as the one I was using the documentation locally.

I just (painfully) verified that something along these lines
```
.. _reference-xpmdp:
xpmdp
~~~~~
.. program-output:: export PYTHONPATH=../:$PYTHONPATH; python ../ixpeobssim/bin/xpmdp.py --help
   :ellipsis: 1,11
   :shell:
```

is actually working (note the ellipsis to get rid of some of the useless printout and have a more compact representation), and I'll work my way through fixing all occurrences.

**Comment by lucabaldini** [comment]
And there's at least one more instance in the quickstart---I am not it.

**Comment by lucabaldini** [comment]
And one in filtering.

**Comment by lucabaldini** [comment]
Shipped in 29.1.2
